index.html

```diff
@@ -126,14 +123,12 @@ <h1 class="title">Hacktoberfest 2021</h1>
                         <table class="main" width="100%">
                             <tr>
                                 <td style="padding:0px 30px 0px 30px;">
-                                    <h2 style="text-align: center;">Hactoberfest 2021: TechCast Ep - 4</h2>
+                                    <h2 style="text-align: center;">Reminder: Hactoberfest - TechCast, Ep 4</h2>
                                     <p>Dear all,</p>
                                     <p>Seasons Greetings! We're hoping this e-mail finds you in good health and high spirits!</p>
-                                    <p>We are happy to inform you that WiOS is back with another exciting event in collaboration with OSC as a part of the Hacktoberfest.</p>
-                                    <p>Hacktoberfest, in its 8th year , is a month-long celebration of open source software run by DigitalOcean.</p>
-                                    <p>The tremendous response we received for Open-source talks ep-1 gave WiOS an incentive to come up with TechCast ep-4.</p>
-                                    <p style="font-weight: bold;">Are you bored of webinars? Wanna learn something new and innovative?</p>
-                                    <p style="font-weight: bold;"> Set reminders for one of the most knowledgeable talks of the year!</p>
+                                    <p>Thank you all for registering for TechCast, Ep 4, which is an event conducted by WiOS club, in 
```

Comment:
Change 'TechCast, Ep 4' to 'TechCast - Ep 4'.
Remove 'which is an event part'. Most of the people opening emails must be knowing what the Event is.